mavros/src/lib/ftf_frame_conversions.cpp

```diff
@@ -58,6 +58,12 @@ static const auto NED_ENU_R = NED_ENU_Q.normalized().toRotationMatrix();
 static const auto AIRCRAFT_BASELINK_R = AIRCRAFT_BASELINK_Q.normalized().toRotationMatrix();
 static const auto ZEROM3D = Eigen::Matrix3d::Zero();
 
+/**
+ * @brief Auxiliar matrices to Covariance transforms
+ */
+using Affine6dTF = Eigen::Matrix<double, 6, 6>;
+using Affine9dTF = Eigen::Matrix<double, 9, 9>;
```

Comment:
I probably have to, instead of creating a affine 3d transform, apply a Affine3d for each rotation and then build the covariance matrix with the diagonal blocks already rotated - the current output seems that it is just multiplying the matrices but not applying the rotation the main diagonal blocks.